test/test.ts

```diff
@@ -572,10 +591,33 @@ test('is.generator', t => {
 	testType(t, 'generator');
 });
 
+test('is.asyncGenerator', t => {
+	testType(t, 'asyncGenerator');
+
+	const fixture = (async function * () {
```

Comment:
what's the added value of these? here you're not testing whether this is the type or not, but whether the implementation of the type "works as promised".

I think it's about the same as checking if a `string` has a `length` property...

Reply:
I did the same test as for asyncFunction (old L564, new L583). It's to keep the same logic.